mkdocs.yml

```
@@ -0,0 +1,64 @@
+site_name: Terraform AWS Provider - Contributor Guide
+site_description: 'Docsite generated from terraform-provider-aws'
+copyright: Hashicorp
+repo_name: hashicorp/terraform-provider-aws
+repo_url: https://github.com/breathingdust/terraform-provider-aws
+edit_uri: "https://github.com/breathingdust/terraform-provider-aws/tree/master/docs"
+
+nav:
+  - Getting Started: index.md
+  - Prioritization: prioritization.md
+  - Development Environment Setup: development-environment.md
+  - Raising a Pull Request: raising-a-pull-request.md
+  - Issue Reporting and Lifecycle: issue-reporting-and-lifecycle.md
+  - Guides:
+    - Adding a New Resource: add-a-new-resource.md
+    - Adding a New Service: add-a-new-service.md
+    - Adding a New Datasource: add-a-new-datasource.md
+    - Adding a New Region: add-a-new-region.md
+    - Adding Import Support: add-import-support.md
+    - Adding Resource Filtering Support: resource-filtering.md
+    - Adding Resource Name Generation: resource-name-generation.md
+    - Adding Resource Tagging Support: resource-tagging.md
+    - Adding a Tag Resource: adding-a-tag-resource.md
+    - Bugs and Enhancements: bugs-and-enhancements.md
+  - Acceptance Testing:
+    - Running and Writing Acceptance Tests: running-and-writing-acceptance-tests.md
+    - Environment Variables: acc-test-environment-variables.md
+  - Reference:
+    - Changelog Process: changelog-process.md
+    - Data Handling and Conversion: data-handling-and-conversion.md
+    - Dependency Updates: dependency-updates.md
+    - Error Handling: error-handling.md
+    - Retries and Waiters: retries-and-waiters.md
+    - Service Packages Refactor: service-package-pullrequest-guide.md
+  - Core Services: core-services.md
+  - Design and Standards:
+    - Provider Design: provider-design.md
+    - Naming: naming.md
+  - FAQ: faq.md
+
+theme:
+  name: material
+  logo: images/hashicorp.png
+  favicon: images/hashicorp-black.png
+  palette:
+    scheme: hashicorp
+    primary: black
+    accent: blue
+  icon:
+    repo: fontawesome/brands/github
+  font: false
+  features:
+    - navigation.instant
+    - content.tabs.link
+
+plugins:
+  - search:
+      lang:
+        - en
+
+markdown_extensions:
+  - pymdownx.highlight
+  - pymdownx.superfences
+  - pymdownx.inlinehilite
```

I think we should add [Tasklist](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#tasklist) support for situations like [Resource Contributions Guidelines](https://breathingdust.github.io/terraform-provider-aws/raising-a-pull-request/#resource-contribution-guidelines), which isn't rendering quite right at the moment.

```suggestion
  - pymdownx.inlinehilite
  - pymdownx.tasklist
```

---

Actually I'm not crazy about the checkbox format (as they aren't actually checkable). I had deleted those from the other lists but we should discuss.

---

I don't love them either to be honest, so if you're on board, I'm fine with going the other direction and removing the checkboxes.